bundles/http/Citrix/5c20233e-ead7-4385-a3f9-c703f194514b/cd40c0ac-cc06-484d-9b52-e8aee2dfa759.js

```diff
@@ -0,0 +1,134 @@
+const ENDPOINT = "/incidents";
+const LIMIT = 100;
+let moment = library.load("moment-timezone");
+
+integration.define({
+    "synchronizations": [
+        {
+            "name": "UpdateComments",
+            "fullSyncFunction": updateComments,
+            "incrementalSyncFunction": updateComments
+        }
+    ],
+    "model": {
+        "tables": [
+            {
+                "name": "updated_comments",
+                "columns": [
+                    {
+                        "name": "id",
+                        "type": "INTEGER",
+                        "primaryKey": true
+                    },
+                    {
+                        "name": "body",
+                        "type": "STRING",
+                        "length": 100
+                    },
+                    {
+                        "name": "created_at",
+                        "type": "DATETIME"
+                    },
+                    {
+                        "name": "commenter_id",
+                        "type": "INTEGER"
+                    },
+                    {
+                        "name": "user_name",
+                        "type": "STRING"
+
+                    },
+                    {
+                        "name": "user_email",
+                        "type": "STRING"
+                    }
+                ]
+            }
+        ]
+    }
+});
+
+function replaceBodyCharacters(key, value) {
+    let str = value;
+    if (key === "body") {
+        str = str.replace(/<[^>]+>/gm, '');
+        str = str.replace('\n&nbsp;', '');
+    }
+    return str;
+}
+
+async function updateComments({ dataStore, client, latestSynchronizationTime }) {
+    let incidentComments = "";
+    let exit = false;
+    let latestSync = new Date(latestSynchronizationTime);
+    let updatedAt = '?q=updated_at>=' + moment(latestSync).format('YYYY-MM-DD');
+
+    let queryFields = '?q=lastUpdateDate>=' + moment(latestSync).format('YYYY-MM-DD') + '&per_page=' + LIMIT + '&page=';
+
+    if (latestSynchronizationTime === undefined) {
+        queryFields = '?per_page=' + LIMIT + '&page=';
+    }

```

Comment:
Minor: if latestSynchronizationTime is undefined, the above code with calls to 'new Date' and `moment().format` is wasted. I'd place it to the `else` branch.